eng/build.sh

```diff
@@ -224,22 +212,24 @@ if [[ "$__NativeBuild" == 1 ]]; then
 fi
 
 #
-# Copy the native SOS binaries to where these tools expect for testing
+# Managed build
 #
 
-if [[ "$__NativeBuild" == 1 || "$__Test" == 1 ]]; then
-    __targetFramework=net8.0
-    __dotnet_sos=$__RootBinDir/bin/dotnet-sos/$__BuildType/$__targetFramework/publish/$__TargetRid
-    __dotnet_dump=$__RootBinDir/bin/dotnet-dump/$__BuildType/$__targetFramework/publish/$__TargetRid
-
-    mkdir -p "$__dotnet_sos"
-    mkdir -p "$__dotnet_dump"
+if [[ "$__ManagedBuild" == 1 ]]; then
 
-    cp "$__BinDir"/* "$__dotnet_sos"
-    echo "Copied SOS to $__dotnet_sos"
+    # __CommonMSBuildArgs contains TargetOS property
+    echo "Commencing managed build for $__BuildType in $__RootBinDir/bin"
+    "$__RepoRootDir/eng/common/build.sh" \
+        --configuration "$__BuildType" \
+        /p:TargetArch="$__TargetArch" \
+        /p:TargetRid="$__OutputRid" \
```

Comment:
```suggestion
        /p:TargetRid="$__TargetRid" \
```

Reply:
Actually - the scripts do the rid calculation inside. It's complicated - we don't want target rid precisely passed down if I recall correctly. This might need some scrutiny in musl